test/features/MainBanner.ts

```diff
@@ -10,6 +10,15 @@ const expectEmitsCloseEvent = async ( wrapper: VueWrapper<any> ): Promise<any> =
 	expect( wrapper.emitted( 'bannerClosed' )[ 0 ][ 0 ] ).toEqual( new CloseEvent( 'MainBanner', CloseChoices.Close ) );
 };
 
+const expectEmitsCloseEventWhenRemainingImpressionsAreZero = async ( wrapper: VueWrapper<any> ): Promise<any> => {
```

Comment:
For banners with the soft close feature I'd prefer using `SoftClose.expectDoesNotShowSoftCloseOnFinalBannerImpression`, either additionally to this or instead of this. The features are testing slightly different things, so I'd add both.

Reply:
I added the `SoftClose.expectDoesNotShowSoftCloseOnFinalBannerImpression` to all tests